commands/htmlCapture.js

```diff
@@ -0,0 +1,48 @@
+
+var fs = require('fs');
+var mkdirp = require('mkdirp');
+
+var mkdirpSync = function(path) {
+    try {
+        mkdirp.sync(path);
+    } catch(e) {
+        if ( e.code != 'EEXIST' ) throw e;
+    }
+}
+
+exports.command = function(fileName, callback) {
+    if (fileName && typeof fileName !== 'string') {
+        throw new Error('htmlCapture expects first parameter to be string; ' + typeof (fileName) + ' given');
+    }
+
+    var htmlCheck = /^\w+\.html$/;
+
+    if (!htmlCheck.test(fileName)){
+        throw new Error('htmlCapture expects first parameter to be camelCased alphanumeric string ending with ".html"');
+    }
+
+    var client = this;
+    var filePath = "tests/fixtures/";
+    var fileLocation = filePath + fileName;
```

Comment:
This should be tested/modified to eliminate the possibility of the folder being created somewhere other than intended.